argo_prod/solrconfig.xml

```diff
@@ -31,16 +31,86 @@
       <str name="fl"> *, score</str>
 
       <str name="qf">
-        id
-        descriptive_tiv
+        sw_display_title_tesim^5
```

Comment:
this qf is overridden in blacklight_config_helper, but it's useful to have the same values here in case running a search directly against Solr is done for debugging or whatever.